source/dubregistry/scheduler.d

```diff
@@ -123,18 +126,20 @@ class PersistentScheduler {
 	}
 
 	private void onTimerFired(string name)
-	{
+	nothrow {
 		auto pevt = name in m_events;
 		if (!pevt || !pevt.handler) return;
 
-		auto now = Clock.currTime(UTC());
+		SysTime now;
```

Comment:
Consider using `MonoTime` for timers. Debugging why timers fire at different time then the intended timeout, due to change of the system time due to NTP is not as fun as it sounds ;)

Reply:
In this case, since the events are supposed to be triggered at a particular `SysTime`,  it it actually the right function, but the whole timer logic in eventcore/vibe:core also still uses `SysTime`, which it really shouldn't.

Reply:
:+1: